app/models/booking.rb

```diff
@@ -0,0 +1,6 @@
+class Booking < ApplicationRecord
+  belongs_to :session
+  belongs_to :user
+
+  validates :canceled, inclusion: { in: [true, false] }
+end
```

Comment:
Can you create a canceled? instance method that returns the value of canceled ?
That's just to make it more understandable that it returns a boolean when used in the code

Reply:
done